react-app/src/components/DatePicker/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import { isDateStringValid, getCurrentDateString } from 'utils/date';
+
+import styles from './index.module.scss';
+
+export default class DatePicker extends PureComponent {
+  static propTypes = {
+    /**
+     * The initial date to show
+     */
+    date: PropTypes.string,
+  };
+
+  static defaultProps = { date: getCurrentDateString() }
+
+  constructor(props) {
+    super(props);
+
+    this.state = { datePicked: this.props.date };
+  }
+
+  _handleChange = event => {
+    const newDatePicked = event.target.value;
+    if (isDateStringValid(newDatePicked)) {
+      this.setState({ datePicked: newDatePicked });
+    }
+  }
+
+  getDate = () => this.state.datePicked;
+
+  render() {
+    const maxDate = getCurrentDateString();
+
+    return (
+      <div>
+        <label className={styles.label}>
+          When?
+        </label>
+        <input
+          type="date"
+          max={maxDate}
+          value={this.state.datePicked}
+          onChange={this._handleChange}
+        />
```

Comment:
I'm perfectly happy with this as the first iteration, but we should use an actual date picker here and not just an input eventually.

Reply:
In mobile, at least in Android, is rendered as a DatePicker (the input acts as a button and opens a calendar to pick the date). **I don't know if this works in iOS...**

We could check this and see if we have to change it or not, what do you think?

Reply:
> We could check this and see if we have to change it or not, what do you think?

Nah, if it works on android, it's good enough for now.

Reply:
Ok